examples/random/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# This Dockerfile builds an image for a client_golang example.
+#
+# Use as (from the root for the client_golang repository):
+#    docker build -f examples/$name/Dockerfile -t prometheus/golang-example-$name .
+
+# The base image contains go, client_golang and client_golang dependencies to
```

Comment:
I'd only use two images, the builder image to build everything and the final `from scratch` image.
Docker should still reuse pre-existing layers.